### PR TITLE
avoid overloading the term ecosystem

### PR DESCRIPTION
**Pull request by oraNod**
* rename the site title
* make purpose clear in the index
* move ecosystem to a secondary page

### CODE REVIEW

**Comment by oraNod** [comment]
@samccann hey, can you take a quick look? this changes things up a little and creates an `ecosystem.html` page in the left nav so it's not directly in the index. that will make it easier to set up the redirect.

the thing I don't like now is that the site title is duplicated in the left nav. there might be a way to fix that but I'd welcome any suggestions. cheers.

File: docs/index.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
 
-# Ansible ecosystem
+Here you can find `How To` guides and `tutorials` for cross-project use cases within the Ansible ecosystem in this docsite.
+
+If you're looking for documentation related to a specific Ansible project, visit the [ecosystem page](ecosystem.md).
+You can explore Ansible projects and find links to the related documentation.
+
+## Extending Ansible automation
+
+Ansible is a programming language and runtime plus an entire ecosystem of projects that extending automation capabilities to a virtually unlimited set of use cases.
```
Comment (samccann) [inline docs/index.md]:
```suggestion
Ansible is a programming language and runtime plus an entire ecosystem of projects that extends automation capabilities to a virtually unlimited set of use cases.
```

Comment (samccann) [reply docs/index.md]: I'm unsure about the rest of the wording on this one. I don't know if we call Ansible a programming language and runtime in other places?  Also, I'd change 'virtually unlimited' to something less grand like 'an extensive and extendable" set of use cases.

**Review by samccann — APPROVED** [review]
LGTM

**Pull request merged**